app/rnx2rtkp/rnx2rtkp.c

```diff
@@ -67,6 +68,10 @@ static const char *help[]={
 " -s sep    field separator [' ']",
 " -r x y z  reference (base) receiver ecef pos (m) [average of single pos]",
 "           rover receiver ecef pos (m) for fixed or ppp-fixed mode",
+" -rr mode  rover position mode (1:average of single pos, 2:read from file,",
```

Comment:
what is the use case of `-rr`? I think it makes sense to add `-rb`, but i don't know where we need to use `-rr`.

Reply:
My reasoning was that the existing `-r` option sets both `prcopt.refpos` and `prcopt.rovpos` to `POSOPT_POS`, so there must be some use case that I am not aware of where it is desirable to change the value of `prcopt.rovpos`.

But I am fine with removing it if you don't think that it brings any value.

Reply:
If it is not used in the WL fixing feature, i think we can remove it.

Reply:
Done